std_msgs/Header header
float32[] q_control_points
float32[] t_control_points